pkgs/tools/graphics/epub-thumbnailer/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, python3Packages, fetchFromGitHub}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epub-thumbnailer";
+  version = "unstable-2022-04-22";
+
+  src = fetchFromGitHub {
+    owner = "marianosimone";
+    repo = pname;
+    rev = version;
+    sha256 = "0mv0zd74g4xfi2pvl8k0z6qvf1987lwk47ia8cyr51capk95p4rp";
+  };
+
+  # The installer is a simple python script, so no tests nor build steps are necessary
+  dontBuild = true;
+  doCheck = false;
+
+  propagatedBuildInputs = with python3Packages; [ pillow ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 src/epub-thumbnailer.py $out/bin/epub-thumbnailer
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/marianosimone/epub-thumbnailer";
+    description = "Script to extract the cover of an epub book and create a thumbnail for it";
+    license = licenses.unlicense;
```

```suggestion
    license = licenses.gpl3Plus;
```



The license is `gpl3Plus` according to

https://github.com/marianosimone/epub-thumbnailer/blob/master/src/epub-thumbnailer.py

---

well observed.

---

Fixed!